packages/gatsby/src/redux/actions/internal.ts

```diff
@@ -249,6 +250,12 @@ export const queryStart = (
   }
 }
 
+export const clearDirtyQueriesListToEmitViaWebsocket = (): IQueryClearDirtyQueriesListToEmitViaWebsocket => {
+  return {
+    type: `QUERY_CLEAR_DIRTY_QUERIES_LIST_TO_EMIT_VIA_WEBSOCKET`,
+  }
+}
+
```

Comment:
Java Script Edition :) But at least it's hopefully clear that this is has one purpose and it's not safe to use it for anything else (as websocket do consume this)